src/planner/ompl/CRRT.cpp

```diff
@@ -165,83 +198,102 @@ void CRRT::freeMemory(void) {
 }
 
 //=============================================================================
-::ompl::base::PlannerStatus
-CRRT::solve(const ::ompl::base::PlannerTerminationCondition &_ptc) {
+::ompl::base::PlannerStatus CRRT::solve(
+    const ::ompl::base::PlannerTerminationCondition& _ptc)
+{
   checkValidity();
-  ::ompl::base::Goal *goal = pdef_->getGoal().get();
-  ::ompl::base::GoalSampleableRegion *goalSampleable =
-      dynamic_cast<::ompl::base::GoalSampleableRegion *>(goal);
+  ::ompl::base::Goal* goal = pdef_->getGoal().get();
+  ::ompl::base::GoalSampleableRegion* goalSampleable
+      = dynamic_cast<::ompl::base::GoalSampleableRegion*>(goal);
 
-  while (const ::ompl::base::State *st = pis_.nextStart()) {
-    Motion *motion = new Motion(si_);
+  while (const ::ompl::base::State* st = pis_.nextStart())
+  {
+    Motion* motion = new Motion(si_);
     si_->copyState(motion->state, st);
     mStartTree->add(motion);
   }
 
-  if (mStartTree->size() == 0) {
+  if (mStartTree->size() == 0)
+  {
     return ::ompl::base::PlannerStatus::INVALID_START;
   }
 
   if (!mSampler)
     mSampler = si_->allocStateSampler();
 
-  Motion *solution = nullptr;
-  Motion *approxsol = nullptr;
+  Motion* solution = nullptr;
+  Motion* approxsol = nullptr;
   double approxdif = std::numeric_limits<double>::infinity();
-  Motion *rmotion = new Motion(si_);
-  ::ompl::base::State *rstate = rmotion->state;
-  ::ompl::base::State *xstate = si_->allocState(); /* temp state */
+  Motion* rmotion = new Motion(si_);
+  ::ompl::base::State* rstate = rmotion->state;
+  ::ompl::base::State* xstate = si_->allocState(); /* temp state */
 
   bool foundgoal = false;
-  while (_ptc == false) {
+  while (_ptc == false)
+  {
     /* sample random state (with goal biasing) */
-    if (goalSampleable && mRng.uniform01() < mGoalBias &&
-        goalSampleable->canSample())
+    if (goalSampleable && mRng.uniform01() < mGoalBias
+        && goalSampleable->canSample())
       goalSampleable->sampleGoal(rstate);
     else
       mSampler->sampleUniform(rstate);
 
     // Continue on invalid sample
-    if (!si_->isValid(rstate)) {
+    if (!si_->isValid(rstate))
+    {
       continue;
     }
 
     /* find closest state in the tree */
-    Motion *nmotion = mStartTree->nearest(rmotion);
+    Motion* nmotion = mStartTree->nearest(rmotion);
 
     /* Perform a constrained extension */
     double bestdist = std::numeric_limits<double>::infinity();
-    Motion *bestmotion = constrainedExtend(_ptc, mStartTree, nmotion, rmotion->state, xstate, goal,
-                                           false, bestdist, foundgoal);
-    if (foundgoal) {
+    Motion* bestmotion = constrainedExtend(
+        _ptc,
+        mStartTree,
+        nmotion,
+        rmotion->state,
+        xstate,
+        goal,
+        false,
+        bestdist,
+        foundgoal);
```

Comment:
I deeply understand your point. Well, we have `// clang-format on [off]` option at least. :sweat: